<?xml version="1.0"?>
<launch>
  <node name="torque_off" pkg="cable_reel" type="torque_off" output="screen">
  </node>

</launch>